### ISSUE TITLE
Consider improved video object tracking

### ISSUE THREAD

**Issue by mzur** [biigle/core#646]
Today I saw FAIR's [co-tracker](https://github.com/facebookresearch/co-tracker) which looks like an impressive video object tracking method. While it has a complicated license, there is also [OmniMotion](https://omnimotion.github.io/) with an Apache license, looking similarly good. We should explore these (GPU-backed) tracking methods to improve the tracking performance of the video annotation tool.

**Comment by dlangenk** [comment]
Both look very good. Also there is [MMtracking](https://github.com/open-mmlab/mmtracking). However it only implements few algorithms.

**Comment by mzur** [comment]
Right, MMTracking would work well with the existing dependencies of MAIA.

**Comment by dlangenk** [comment]
I had a very small test with co-tracker tracking a fish in a video with good visibility. It worked well until the fish went into the corals and emerged again at a slightly different location. The tracker got stuck like most other trackers also did in previous tests.

**Comment by mzur** [comment]
Maybe #908 may solve this.

**Comment by mzur** [comment]
Closing in favor of #908
